src/charts/tooltip/TooltipComponent.js

```diff
@@ -6,6 +6,8 @@ import tooltip from './tooltipChart';
 export default class TooltipComponent extends React.Component {
 
     static PropTypes = {
+        children: PropTypes.node.isRequired,
```

Comment:
You can do this instead: https://reactjs.org/docs/typechecking-with-proptypes.html#requiring-single-child

Which also has the added benefit of making sure there's only one child.